docs/tasks/argo.rst

```diff
@@ -0,0 +1,283 @@
+Automating ActionSet Creation using Argo Cron Workflows
+**********************************************************
+
+Argo Workflows enable us to schedule operations. In the Kanister project,
+Argo Cron Workflows will be used to automate the creation of ActionSets to
+execute Blueprint actions at regular intervals.
+
+To summarize, ActionSets are CRDs that are used to execute actions
+from Blueprint CRDs. The Kanister controller watches for the creation
+of ActionSets and executes the specified action.
+
+In this tutorial, you will schedule the creation of a backup ActionSet using
+Argo Cron Workflows.
+
+
+Prerequisites
+=============
+
+* Kubernetes ``1.20`` or higher.
+* A running Kanister controller in the ``Kanister`` namespace. See :ref:`install`
+* ``kanctl`` CLI installed. See Tools_.
+
+Architecture
+============
+
+.. image:: img/argo-cron-architecture.png
+
+Steps
+=====
+
+
+Step 1 - Setup Argo
+--------------------
+
+Download the Argo CLI from their Releases_ page.
+
+Create a separate namespace for the Workflows.
+
+.. code-block:: bash
+
+  kubectl create ns argo
+
+For this tutorial, the minimal manifest file will be used to download
+Argo CRDs and setup their resources.
```

Comment:
Yes I wanted to add the direct link in the `kubectl apply` command but one might have to modify the managed namespace configuration in this file to make things work. So it might be easy to debug if the file exists locally.